cttportfolio.css

```diff
@@ -3,168 +3,481 @@
 }
 html{
 	font-size:62.5%;
+	font-family:'Montserrat', sans-serif;
+	color:rgba(4,8,9,1);
 }
 body{
 	margin:0;
+	background-color:rgb(2,8,9);
 }
 main > div{
 	margin:0 auto;
-	max-width:1350px;
-	padding:0 1%;
+	max-width:1300px;
+}
+main{
+}
+h4, h5, h6,.pp_body h1{
+	font-weight:700;
+	font-size:6.0rem;
+	text-transform:uppercase;
+    -webkit-text-stroke-width: 0.3px;
+    -webkit-text-stroke-color: black;
+	color: rgb(248,248,255);
+}
+h5{
+	font-weight:600;
+	font-size:4rem;
+}
+.am_text,.aboutme img, button, .navlinks a, .pf_sub_links img, .footer_icon_size > a > img, .contactmetext > div{
```

Comment:
This line is rather long, is there a way you could split this up a bit?? (Not sure if possible).

Reply:
yeah i could add a class the every element that I want to apply this too but idk if its better tbh